kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -161,6 +163,11 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
 
   def getOpenSessionCount: Int = handleToSession.size()
```

Comment:
Does this method still make sense? If not, I think we can remove it to avoid it being used incorrectly.

Reply:
addressed